src/ui/public/copee/copee.js

```diff
@@ -0,0 +1,46 @@
+import Notifier from 'ui/notify/notifier';
+import uiModules from 'ui/modules';
+// borrowed heavily from https://github.com/styfle/copee
+
+let module = uiModules.get('kibana');
+
+module.service('copee', function () {
+  const copee = this;
+
+  copee.urlToClipboard = (url, name) => {
+    const notify = new Notifier({
+      location: `Share ${name}`,
+    });
+
+    const a = document.createElement('a');
+    a.style.color = 'transparent';
+    a.style.border = 'none';
+    document.body.appendChild(a);
```

Comment:
I don't think you even need to append the anchor to the document, the href seems to get generated correctly without it.